editor/editor_export_godot3.cpp

```diff
@@ -480,15 +464,12 @@ static const char *prop_renames[][2] = {
 };
 
 static const char *type_renames[][2] = {
-	{ "SpatialPlayer", "Spatial" },
-	{ "SpatialSamplePlayer", "Spatial" },
-	{ "SpatialStreamPlayer", "Spatial" },
-	{ "Particles", "Spatial" },
-	{ "SamplePlayer", "Node" },
-	{ "SamplePlayer2D", "Node2D" },
-	{ "SoundPlayer2D", "Node2D" },
-	{ "StreamPlayer2D", "Node2D" },
-	{ "Particles2D", "Node2D" },
+	{ "StreamPlayer", "AudioStreamPlayer" },
+	{ "SpatialSamplePlayer", "AudioStreamPlayer3D" },
+	{ "SpatialStreamPlayer", "AudioStreamPlayer3D" },
+	//{ "SamplePlayer", "Node" },
```

Comment:
I think the main purpose of those Node/Node2D conversion is so that the actual node information is not lost. Obviously Node/Node2D is wrong, but then in your scene you will see a Node named "SamplePlayer" and noticed that it needs fixing :)